scripts/scripts-1.js

```diff
@@ -0,0 +1,174 @@
+// Створити функцію-конструктор Людини. Властивості, які будуть описувати екземпляр Людини - ім'я, вік, стать, національність, країна, список країн для подорожей. Створити універсальні функції - знайомство, прокидатися, засинати й список країн, які Людина хоче відвідати. Роботу цих функцій можна виводити через console.log. Але в кожній з функцій повинно використовуватись якомога більше характеристик конкретної Людини на якій ця функція викликається (підказка: тут треба використати call/apply)
+
+//Шаблон
+
+function Human(name, age, sex, nationality, country, countryWishList) {
+  this.name = name;
+  this.age = age;
+  this.sex = sex;
+  this.nationality = nationality;
+  this.country = country;
+  this.countryWishList = countryWishList;
+}
+
+//Приклади
+
+const Kris = new Human ('Kris', 22, 'female', 'american', 'USA', 'Ukraine, Japan, Peru, Kenya, Island, Spain',)
```

Comment:
The description - list of countries should be an array

```suggestion
const Kris = new Human ('Kris', 22, 'female', 'american', ['USA', 'Ukraine, Japan, Peru, Kenya, Island, Spain'])
```